patterns/2-structured/innersource-customer-interview-questions.md

```diff
@@ -0,0 +1,74 @@
+## Title
+
+InnerSource Customer Interview Questions
+
+## Patlet
+
+An organization has decided to create an InnerSource program and need a place to start. Using a customer interview will help evaluate InnerSource pain points across the organization.
+
+## Problem
+
+InnerSource can address a number of areas that impact collaboration. The problem is determining what organizational pain points to address first. People want every pain point that InnerSource can address, solved fast. Evaluating team needs can be ambiguous if the conversation is not structured around an InnerSource framework.
+
+## Context
```

Comment:
I recommend to format both Context and Forces with bullet points.
That makes it easier for readers to check these one-by-one to confirm if these points apply to them.

Also check Context and Forces for possible duplication between the two sections.

Reply:
I would still recommend to do this.

However now that this pattern is in status "Initial", we can work this out some time later.
Keeping this thread open for reference.